conf.js

```diff
@@ -1,9 +1,24 @@
+var HtmlReporter = require('protractor-beautiful-reporter');
+var today = new Date(),
+timeStamp = '0' + today.getDate() + '-' + (today.getMonth() + 1) + '-' + today.getFullYear() + '--' + today.getHours() + 'h-' + today.getMinutes() + 'm';
+
 exports.config = {
   framework: 'jasmine',
   seleniumAddress: 'http://localhost:4444/wd/hub',
   specs: ['./tests/tests.js'],
   multiCapabilities: [{
-    browserName: 'firefox'
+    browserName: 'chrome'
   }],
-  jasmineNodeOpts: {defaultTimeoutInterval: 60000}
+  jasmineNodeOpts: {defaultTimeoutInterval: 60000},
+  onPrepare: () => {
+    jasmine.getEnv().addReporter(new HtmlReporter({
+      baseDirectory: 'reports/'+timeStamp,

```

Comment:
The reports folder structure doesn't comply with the required specification.
![image](https://user-images.githubusercontent.com/56347382/68399594-4b0f7500-017f-11ea-9cf3-7e1e5a48dc4a.png)

Reply:
The timestamp was updated to the desired format.